Pocket/pocket-ios
# feat(reader actions): reader actions in webview [IN-733]

## Summary
Implemented reader actions in Webview

## References 
[IN-733](https://getpocket.atlassian.net/browse/IN-733)

## Implementation Details
Reader activity options are implemented by using a custom class ReaderActionsWebActivity (UIKit - UIActivity). 

## Test Steps
when I view Home and it shows recommendations.
then I should be able to click on Recommendation, that opens in Webview and clicking on 'Share' options should show a list of Activities and list should have custom activities - 'Save' & 'Report'

when I view Home and it shows recommendations.
then I should be able to click on Recommendation that is saved already, that opens in Webview and clicking on 'Share' options should show a list of Activities and list should have custom activities - 

when I view Home and it shows recommendations.
then I should be able to click on Recommendation that is archived already, that opens in Webview and clicking on 'Share' options should show a list of Activities and list should have custom activities - 

## PR Checklist:

## Screenshots
| Recommendation        | Recommendation After Save           | Recommendation After Archive  |
|-------------|-------------|-----|
| ![Simulator Screen Shot - iPhone 14 - 2022-11-03 at 12 06 21](https://user-images.githubusercontent.com/110417980/199776519-8cb9d160-8fe2-4ebd-9d59-2c055f613a45.png) | ![Simulator Screen Shot - iPhone 14 - 2022-11-03 at 12 42 57](https://user-images.githubusercontent.com/110417980/199782083-174de876-f3fa-448a-9bd2-251a4b1c7d80.png) | ![Simulator Screen Shot - iPhone 14 - 2022-11-03 at 12 43 53](https://user-images.githubusercontent.com/110417980/199782349-f157e488-9f27-4391-82e1-56a30ebf22de.png) |

Video:

https://user-images.githubusercontent.com/110417980/199789140-e67f5c36-39da-475c-b3e7-0dc9d6605e31.mp4
